### PR TITLE
research page modified

### PR DESCRIPTION
**Pull request by mharshita**
@anushbhatia  @smaranjitghose  I have made all the modifications in the research page. Please review. 
There is just one bug in the cite button, that when it is clicked, vancouver and bibtex of all the papers are showing up. Please give some time to fix it.

### CODE REVIEW

File: assets/js/research.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+const researchTable = document.querySelector(".main");
+const citeTable = document.querySelector(".citeT");
+
+const research = [
+    {
+        title : "A Batch Normalized Inference Network Keeps the KL Vanishing Away",
+        authors : "Qile Zhu, Wei Bi, Xiaojiang Liu, Xiyao Ma, Xiaolin Li and Dapeng Wu",
+        conferences : "The 58th Annual Meeting of the Association for Computational Linguistics",
+        year : ", 2020",
+        researchYr : "2 Jan, 2020",
```
Comment (smaranjitghose) [inline assets/js/research.js:12]: This is not required. Only year is sufficient

File: assets/js/research.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    },
+
+    {
+        title : "A Comprehensive Analysis of Preprocessing for Word Representation Learning in Affective Tasks",
+        authors : "Nastaran Babanejad, Ameeta Agrawal, Aijun An and Manos Papagelis",
+        conferences : "The 58th Annual Meeting of the Association for Computational Linguistics",
+        year : ", 2020",
+        researchYr : "2 Jan, 2020",
+        image : "assets/images/research-page/wordRepresentation.png",
+        vancouver : "Nastaran Babanejad, Ameeta Agrawal, Aijun An and Manos Papagelis. A Comprehensive Analysis of Preprocessing for Word Representation Learning in Affective Tasks. The 58th Annual Meeting of the Association for Computational Linguistics 2020.",
```
Comment (smaranjitghose) [inline assets/js/research.js:36]: Nice work with this!

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
research_page
# Some suggestions:
- In the cards, the year of the work is sufficient
- You can put the year below the authors and conference name
- Shift the cite button to the right

The remaining UI looks superb!❤ Nice work @mharshita .
There's a small bug in the Cite Button, Fix it in a later PR

File: assets/js/research.js (hunk 1 above)
Comment (mharshita) [reply assets/js/research.js:12]: Okay sir

File: assets/js/research.js (hunk 2 above)
Comment (mharshita) [reply assets/js/research.js:36]: thanks :)

**Comment by mharshita** [comment]
@smaranjitghose  Okay!

**Comment by mharshita** [comment]
@smaranjitghose sir i have made the required changes. please review.

**Pull request merged**